java/src/jmri/Sensor.java

```diff
@@ -65,12 +65,12 @@
     /**
      * Request a call-back when the bound KnownState property changes.
      */
-    public void addPropertyChangeListener(@Nullable java.beans.PropertyChangeListener l);
+    public void addPropertyChangeListener(@CheckForNull java.beans.PropertyChangeListener l);
 
     /**
      * Remove a request for a call-back when a bound property changes.
      */
-    public void removePropertyChangeListener(@Nullable java.beans.PropertyChangeListener l);
+    public void removePropertyChangeListener(@CheckForNull java.beans.PropertyChangeListener l);
```

Comment:
java.beans.PropertyChangeSupport allows a null listener to be passed in the addPropertyChangeListener and removePropertyChangeListener methods (it silently ignores it, but the [JavaDocs](http://docs.oracle.com/javase/8/docs/api/java/beans/PropertyChangeSupport.html#addPropertyChangeListener-java.beans.PropertyChangeListener-) explicitly define how null values are handled). The JavaBeans specification does not define how null listeners are to be handled.